### PR TITLE
docs: fix typos in recent blogs

### PR DESCRIPTION
**Pull request by yzeng25**
Changes:

Found some typos in recent blog posts when syncing to Medium account. This pull request will fix these typos.

### CODE REVIEW

File: website/blog/2022/03/04/apisix-uses-coredns-enable-service-discovery.md [hunk 1]
```diff
@@ -124,9 +124,9 @@ CoreDNS listens on port `53` by default, and will read the `Corefile` configurat
 
 `Corefile` mainly implements functions by configuring plugins, so we need to configure three plugins:
 
-- `hosts`： You can use this parameter to bind the service name and IP address. fallthrough means that when the current plugin cannot return normal data, the request can be forwarded to the next plugin for processing (if it exists).
-- `forward`： Indicates to proxy the request to the specified address, usually the authoritative DNS server address.
-- `log`： Don't configure any parameters to print log information to the console interface for debugging.
+- `hosts`:  You can use this parameter to bind the service name and IP address. fallthrough means that when the current plugin cannot return normal data, the request can be forwarded to the next plugin for processing (if it exists).
```
Comment (SkyeYoung) [inline website/blog/2022/03/04/apisix-uses-coredns-enable-service-discovery.md]:
```suggestion
- `hosts`: You can use this parameter to bind the service name and IP address. Fallthrough means that when the current plugin cannot return normal data, the request can be forwarded to the next plugin for processing (if it exists).
```

File: website/blog/2022/03/04/apisix-uses-coredns-enable-service-discovery.md [hunk 2]
```diff
@@ -124,9 +124,9 @@ CoreDNS listens on port `53` by default, and will read the `Corefile` configurat
...
+- `forward`:  Indicates to proxy the request to the specified address, usually the authoritative DNS server address.
```
Comment (SkyeYoung) [inline website/blog/2022/03/04/apisix-uses-coredns-enable-service-discovery.md]:
```suggestion
- `forward`: Indicates to proxy the request to the specified address, usually the authoritative DNS server address.
```

File: website/blog/2022/03/04/apisix-uses-coredns-enable-service-discovery.md [hunk 3]
```diff
@@ -124,9 +124,9 @@ CoreDNS listens on port `53` by default, and will read the `Corefile` configurat
...
+- `log`:  Don't configure any parameters to print log information to the console interface for debugging.
```
Comment (SkyeYoung) [inline website/blog/2022/03/04/apisix-uses-coredns-enable-service-discovery.md]:
```suggestion
- `log`: Don't configure any parameters to print log information to the console interface for debugging.
```

**Review by SkyeYoung — CHANGES_REQUESTED** [review]
We may also need a typo check. :D

But it can be a difficult thing. 🤔

File: website/blog/2022/03/04/apisix-uses-coredns-enable-service-discovery.md (hunk 1 above)
Comment (yzeng25) [reply website/blog/2022/03/04/apisix-uses-coredns-enable-service-discovery.md]: Fixed.

File: website/blog/2022/03/04/apisix-uses-coredns-enable-service-discovery.md (hunk 2 above)
Comment (yzeng25) [reply website/blog/2022/03/04/apisix-uses-coredns-enable-service-discovery.md]: Fixed.

File: website/blog/2022/03/04/apisix-uses-coredns-enable-service-discovery.md (hunk 3 above)
Comment (yzeng25) [reply website/blog/2022/03/04/apisix-uses-coredns-enable-service-discovery.md]: Fixed.

**Comment by yzeng25** [comment]
> We may also need a typo check. :D
> 
> But it can be a difficult thing. 🤔

Good point.

File: website/blog/2022/02/10/splunk-apisix-integration.md [hunk 4]
```diff
@@ -37,7 +37,7 @@ This article explains how to configure and use the [Splunk HEC](https://docs.spl
 
 [Splunk HTTP Event Collector (HEC)](https://docs.splunk.com/Documentation/Splunk/latest/Data/UsetheHTTPEventCollector)is an HTTP event collector provided by Splunk that provides the ability to send data and application events to Splunk using the HTTP(S) protocol.
```
Comment (SylviaBABY) [inline website/blog/2022/02/10/splunk-apisix-integration.md]:
```suggestion
[Splunk HTTP Event Collector(HEC)](https://docs.splunk.com/Documentation/Splunk/latest/Data/UsetheHTTPEventCollector)is an HTTP event collector provided by Splunk that provides the ability to send data and application events to Splunk using the HTTP(S) protocol.
```

File: website/blog/2022/02/10/splunk-apisix-integration.md [hunk 5]
```diff
@@ -37,7 +37,7 @@ This article explains how to configure and use the [Splunk HEC](https://docs.spl
...
 
-## About splunk-hec-logging lugin
+## About splunk-hec-logging plugin
 
 The [splunk-hec-logging](https://github.com/apache/apisix/blob/master/docs/en/latest/plugins/splunk-hec-logging.md) is used to forward Apache APISIX request logs to Splunk for analysis and storage. When enabled, Apache APISIX will take the request context information during the Log phase, serialize it into [Splunk Event Data Format](https://docs.splunk.com/Documentation/Splunk/latest/Data/FormateventsforHTTPEventCollector#Event_metadata) and submit it to the batch queue. The data in the queue is committed to Splunk HEC when the maximum processing capacity of the batch queue per batch is triggered, or when the maximum time to refresh the buffer is reached.
```
Comment (EstelleRao) [inline website/blog/2022/02/10/splunk-apisix-integration.md]: 1. What does it mean by saying "take the request context information during the Log phase"? 
My guess is that "Apache APISIX will collect the request context information while tracking or logging the request"?

2. Put the condition or trigger before the thing that follows to make the context more logic and coherent.

```suggestion
When the maximum processing capacity of a queue is reached or when the maximum time to refresh the buffer is reached, data in the queue will be committed to Splunk HEC.
```

**Review by SkyeYoung — DISMISSED** [review]

**Review by juzhiyuan — APPROVED** [review]

File: website/blog/2022/02/10/splunk-apisix-integration.md (hunk 4 above)
Comment (yzeng25) [reply website/blog/2022/02/10/splunk-apisix-integration.md]: Fixed.

File: website/blog/2022/02/10/splunk-apisix-integration.md (hunk 5 above)
Comment (yzeng25) [reply website/blog/2022/02/10/splunk-apisix-integration.md]: Fixed.

**Review by SylviaBABY — APPROVED** [review]

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/02/10/splunk-apisix-integration.md [hunk 6]
```diff
@@ -35,7 +35,7 @@ Apache APISIX 作为一个高性能的 API 网关不仅在性能上有着良好
 
 [Splunk](https://www.splunk.com/) 是一个机器数据的全文搜索引擎，可应用于采集、索引、搜索和分析各种应用数据，根据 [DB Engines 的检索引擎排名](https://db-engines.com/en/ranking/search+engine)，目前 Splunk 位列第二，是一款应用广泛的全文检索软件。Splunk 和 ElasticSearch 一样，是准实时可以提供不间断搜索结果的数据流。
 
-[Splunk HTTP Event Collector (HEC)](https://docs.splunk.com/Documentation/Splunk/latest/Data/UsetheHTTPEventCollector) 是 Splunk 提供的 HTTP 事件收集器，主要提供以 HTTP(S) 协议将数据和应用程序事件发送到 Splunk 的能力。
+[Splunk HTTP Event Collector(HEC)](https://docs.splunk.com/Documentation/Splunk/latest/Data/UsetheHTTPEventCollector) 是 Splunk 提供的 HTTP 事件收集器，主要提供以 HTTP(S) 协议将数据和应用程序事件发送到 Splunk 的能力。
```
Comment (SkyeYoung) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/02/10/splunk-apisix-integration.md:38]: Should this space be removed?🤔

Comment (SkyeYoung) [reply website/i18n/zh/docusaurus-plugin-content-blog/2022/02/10/splunk-apisix-integration.md:38]: https://grammar.yourdictionary.com/punctuation/when-how-use-parentheses-correctly-examples

@yzeng25 We may need another PR to fix this. 😂

Comment (yzeng25) [reply website/i18n/zh/docusaurus-plugin-content-blog/2022/02/10/splunk-apisix-integration.md:38]: either way is fine @SylviaBABY

Comment (yzeng25) [reply website/i18n/zh/docusaurus-plugin-content-blog/2022/02/10/splunk-apisix-integration.md:38]: > 

ok, I agree.

Comment (yzeng25) [reply website/i18n/zh/docusaurus-plugin-content-blog/2022/02/10/splunk-apisix-integration.md:38]: Created a pull request to completely fix this issue https://github.com/apache/apisix-website/pull/1019 @SkyeYoung

**Pull request merged**